### PR TITLE
CVAT UI: batch of fixes

### LINKED ISSUE #1836: CVAT new UI: object's details menu settings
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour

The object's details menu should have a setting to automatically hide or unhide it for all objects, including new.
### Current Behaviour

Object's details are hidden by default. We have to make one more click every time to enter some attributes.
It's inconvenient.
details
details_1

### Possible Solution

Add a setting "always show objects details" into "settings". This setting should be enabled for new objects too.
 It'll let annotate attributes quickly.

### Steps to Reproduce (for bugs)

1.
1.
1.
1.

### Your Environment

- Git hash commit (`git log -1`):
- Docker version `docker version` (e.g. Docker 17.0.05):
- Are you using Docker Swarm or Kubernetes?
- Operating System and version (e.g. Linux, Windows, MacOS):
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

### LINKED ISSUE #1945: Feature Request:  Annotation in process outline color wheel.
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour
To be able to select which color outlines the polygon shape while marking(or other shapes as well).

### Current Behaviour
Is defaulted to black.

### Context
This would be a minor enhancement.  At times during marking, it's beneficial for the annotator to be able to change this outline color while marking.  The reasoning is say the outline is black and you're marking a dark figure, it makes it difficult to verify if you're properly marking.  This of course applies to other colors as well.  I've manually changed this in the past to help annotators with their task, but I think it wouldn't be too difficult to add a color wheel or something to the UI to allow this color to be changed on the fly.

### PR DESCRIPTION
**Pull request by ActiveChooN**
### Motivation and context

Fixes #1945, fixes #1836 

### How has this been tested?

Manually

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
~~- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly~~
~~- [ ] I have added tests to cover my changes~~
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by bsekachev** [comment]
Collapsed logic works not always.
For example when you do not have any objects, press the button "expand all" and then create an object, the object is collapsed

**Comment by ActiveChooN** [comment]
> Collapsed logic works not always.

@bsekachev, fixed.

**Review by bsekachev — APPROVED** [review]

**Comment by bsekachev** [comment]
Version conflicts appeared

**Review by bsekachev — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Ability to configure email verification for new users (<https://github.com/opencv/cvat/pull/1929>)
 - Link to django admin page from UI (<https://github.com/opencv/cvat/pull/2068>)
 - Notification message when users use wrong browser (<https://github.com/opencv/cvat/pull/2070>)
+- Annotation in process outline color wheel (<https://github.com/opencv/cvat/pull/2084>)
```
Comment (nmanovic) [inline CHANGELOG.md:36]: @bsekachev , @ActiveChooN , could you please fix CHANGELOG?

Comment (bsekachev) [reply CHANGELOG.md:36]: Fixed in develop

**Pull request merged**